snap/snapcraft.yaml

```diff
@@ -0,0 +1,55 @@
+name: gateway
+version: git
+summary: Things Gateway.
+description: Web of Things gateway.
+
+confinement: strict
+grade: stable
+base: core18
+
+apps:
+  gateway:
+    command: env TMPDIR=$XDG_RUNTIME_DIR desktop-launch $SNAP/gateway
+    plugs:
+      - desktop
+      - desktop-legacy
+      - home
+      - x11
+      - browser-support
+      - network
+      - gsettings
+      - pulseaudio
+      - opengl
```

Comment:
What are all these desktop plugs needed for? The gateway is a headless command line application. X11 shouldn't be necessary.

Reply:
Yeah, that's my bad I thought I had removed in a early version. So thank you for catching that I will remove the plus. :+1: